apps/core-api/src/plugins/socket.ts

```diff
@@ -89,9 +91,23 @@ export default fp(
               session.status === 'COMPLETED' // TODO: 완료 상태로 들어올 때 처리
             ) {
               fastify.log.info(
-                `[${sessionId}] Questions are already ready. Notifying re-joined client ${socket.id}.`,
+                `[${sessionId}] Session status is ${session.status}. Notifying client ${socket.id}.`,
               )
-              socket.emit('server:questions-ready', { sessionId })
+              const answeredSteps = await fastify.prisma.interviewStep.findMany(
+                {
+                  where: {
+                    interviewSessionId: sessionId,
+                    answer: { not: null },
+                  },
+                  orderBy: { aiQuestionId: 'asc' },
+                },
+              )
```

Comment:
혹시 이런 형식으로 값을 받을 수 있을까요?
기존 방식을 사용하면 front에서 다시 메인 질문과 꼬리 질문을 `{ main: string; followUps: string[] }`
이런 형식으로 변환해야하는데
DB에서 이렇게 받으면 바로 꼬리질문을 메인 질문에 종속되게 받을 수 있을 것 같아서요!

```suggestion
              const answeredSteps = await fastify.prisma.interviewStep.findMany(
                {
                  where: {
                    interviewSessionId: sessionId,
                    parentStepId: null, // 메인 질문만,
                    answer: { not: null },
                  },
                  orderBy: { aiQuestionId: 'asc' },
                  include: {
                    tailSteps: {
                      where: { answer: { not: null } },
                      orderBy: { aiQuestionId: 'asc' },
                    },
                  },
                },
              )
```

Reply:
감사합니다!
생각해보니 이미 aiQuestionId를 가지고 main과 list를 만드는 로직이 있긴 하네요^^;